app/lib/importers/case_importer.rb

```diff
@@ -19,37 +19,43 @@ def import_cases
 
       casa_case = CasaCase.find_by(case_number: casa_case_params[:case_number], casa_org_id: org_id)
       volunteer_assignment_list = email_addresses_to_users(Volunteer, String(row[:case_assignment]))
+      next_court_date = row[:next_court_date]
 
       if casa_case # Case exists try to update it
         unless casa_case.active
           raise "Case #{casa_case.case_number} already exists, but is inactive. Reactivate the CASA case instead."
         end
 
-        update_casa_case(casa_case, casa_case_params, volunteer_assignment_list)
+        update_casa_case(casa_case, casa_case_params, volunteer_assignment_list, next_court_date)
       else # Case doesn't exist try to create a new case
-        create_casa_case(casa_case_params, volunteer_assignment_list)
+        create_casa_case(casa_case_params, volunteer_assignment_list, next_court_date)
       end
     end
   end
 
   private
 
-  def create_casa_case(case_params, volunteer_assignment_list)
+  def create_casa_case(case_params, volunteer_assignment_list, next_court_date)
     casa_case = CasaCase.new(case_params)
     casa_case.transition_aged_youth = casa_case.in_transition_age?
     casa_case.casa_org_id = org_id
     casa_case.save!
 
+    casa_case.court_dates.create(date: next_court_date)
     casa_case.volunteers << volunteer_assignment_list
 
     casa_case
   end
 
-  def update_casa_case(casa_case, case_params, volunteer_assignment_list)
+  def update_casa_case(casa_case, case_params, volunteer_assignment_list, next_court_date)
     if record_outdated?(casa_case, case_params)
       casa_case.update(case_params)
     end
 
+    if casa_case.next_court_date.nil?
+      casa_case.court_dates.create(date: next_court_date)
+    end
```

Comment:
Note that this will only add `next_court_date` to existing cases that don't have a court date yet. If a case already has other court dates (even ones in the past), this will be ignored. I'm open to changing this functionality, I just wasn't sure if it was needed.